### ISSUE TITLE
Can't reproduce the results in other COCO-format instance segmentation dataset

### ISSUE THREAD

**Issue by Z1zs** [showlab/DatasetDM#25]
Thanks for your great job!
I tried to train a P-Decoder for instance segmentation tasks in another COCO-format dataset and the mask results of P-Decoder seemed very poor. The dataset I used is CIS([Construction Instance Segmentation Dataset](https://github.com/XZ-YAN/CIS-Dataset)) and the total training epochs are 5000. Here is some results of P-Decoder in different stages(epochs), all of them seem to fail in annotating.
<img width="909" alt="CIS P-Deocder" src="https://github.com/showlab/DatasetDM/assets/103265342/5784d0d7-22dd-4671-a834-2680c501887a">
I just adjust the code for dataset loading, and all the training codes remains the same.
Are there extra tricks you used in the training process like warm-up? Thank you in advance!

**Comment by weijiawu** [comment]
Sorry for the delayed response. The performance of instance segmentation is quite poor because instance segmentation is rather challenging. How much data did you use from CIS here?

**Comment by Z1zs** [comment]
thank u very much for the reply! We selected 20 images for each category in training_instances process and the total number of samples for training is 260(13 categories). 
Despite it's poor performance, would there be any possible methods to improve it?
Thx again for your help.

**Comment by weijiawu** [comment]
I think that might be reasonable.

 Perhaps you can try training with more data, as the AP metrics provided in our paper are also very low. Instance segmentation is very challenging when dealing with small datasets.
